modules/bioinspired/src/retina.cpp

```diff
@@ -649,7 +663,7 @@ void RetinaImpl::_convertValarrayBuffer2cvMat(const std::valarray<float> &grayMa
             for (unsigned int j=0;j<nbColumns;++j)
             {
                 cv::Point2d pixel(j,i);
-                outMat.at<unsigned char>(pixel)=(unsigned char)*(valarrayPTR++);
+                outMat.at<unsigned char>(pixel)=(unsigned char)round(*(valarrayPTR++));
```

Comment:
This is to fix accuracy test, because in cpu implementation, final pixel is converted from float to int with truncating conversion,  but in ocl implementation, float to int conversion is rounding conversion.  this change is to make sure we use same conversion method